src/app/events/tracking/HandleParticipantDuringEvent.ts

```diff
@@ -96,14 +153,45 @@ const isChannelActivePOAPEvent = async (
 	return false;
 };
 
-const removeDeafenedUser = async (channelId: string | null, guildId: string | null, userId: string | null) => {
+const removeDeafenedUser = async (channelId: string | null, guildId: string | null, userId: string | null, userTag: string | undefined) => {
 	if (channelId == null || guildId == null || userId == null) {
 		return;
 	}
 	
 	const db: Db = await MongoDbUtils.connect(constants.DB_NAME_DEGEN);
 	const poapParticipantsCol: Collection<POAPParticipant> = db.collection(constants.DB_COLLECTION_POAP_PARTICIPANTS);
-	
+	const addtionalPoapParticipantsCol: Collection<POAPParticipant> = db.collection(constants.DB_COLLECTION_ADDITIONAL_POAP_PARTICIPANTS_LIST);
```

Comment:
These are a lot of db operations for a single user when an event change occurs. Keeping in mind that there may be hundreds of users changing their status, can we condense some of these operations?